pdf_craft/pipeline/epub/translation/translator.py

```
@@ -0,0 +1,194 @@
+                action=submit,
+                payload=_ElementContext(
+                    element_type=_ElementType.CHAPTER,
+                    chapter_data=(chapter_path, xml),
+                ),
+            )
+
+
+def _get_version() -> str:
+    try:
+        return get_package_version("epub-translator")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)